libs/langchain-qdrant/src/vectorstores.ts

```diff
@@ -113,18 +118,18 @@ export class QdrantVectorStore extends VectorStore {
    * from the documents using the `Embeddings` instance and then adds the
    * vectors to the database.
    * @param documents Array of `Document` instances to be added to the Qdrant database.
-   * @param documentOptions Optional `QdrantAddDocumentOptions` which has a list of JSON objects for extra querying
+   * @param customPayloads Optional `QdrantCustomPayload` array which has a list of JSON objects for extra querying.
    * @returns Promise that resolves when the documents have been added to the database.
    */
   async addDocuments(
     documents: Document[],
-    documentOptions?: QdrantAddDocumentOptions
+    customPayloads?: QdrantCustomPayload[],
```

Comment:
This is breaking isn't it?

Would prefer to keep the wrapping

Reply:
Sure, feel free to keep it 🫣 I understand the urge, but in my opinion I'm always for the correct way instead producing the legacy code of tomorrow. It's yet already complicated enough with the customPayloads on document level and the customPayload on dbConfig level.

But I understand that you don't want to have breaking changes, even a simple one like this.

Reply:
Yeah it's a fine line - I'd err towards keeping it for now with a note to change it in the next minor bump.